src/routes/Devilstable.svelte

```diff
@@ -71,45 +64,101 @@
 		}
 		return fractionData;
 	});
+	/** @type {{label: string, values: boolean[]}[]} */
 	let boolData = $derived([
-		...noFractions.map((d) => {
-			/** @type {boolean[]} */ const values = new Array(DATA_MAX).fill(false);
+		...inputChipValueLabels.map((d) => {
+			const info = data.find((c) => c.label === d);
```

Comment:
Consider renaming `d` and `c` (since `c` is referring to `data` and `d` to `chip`, I got confused... actually just flipping the letters would be perfect!).

Reply:
Also consider renaming `info`. I just asked ChatGPT and it suggested me `item` or `entry` as singular-forms of `data`. 

ChatGPT-query: `I have an object array called 'data'. How would I name just one element of it? Since there is no singular/plural of data, it's not that easy. Are there conventions?`

Reply:
> Also consider renaming `info` to make it more explicit. I just asked ChatGPT and it suggested me `item` or `entry` as singular-forms of `data`. (Maybe prepend a `data`, e.g. `data_item`)
> 
> ChatGPT-query: `I have an object array called 'data'. How would I name just one element of it? Since there is no singular/plural of data, it's not that easy. Are there conventions?`

There is always [datum](https://en.wiktionary.org/wiki/datum) but especially in a German-speaking context it may lead to confusion (plus it is marked *dated* in the first sense given).

Reply:
> There is always datum

Oh, that's nice, but I agree with the caveat you make. Another solution would be to find another word for 'data' alltogether, since it is very generic, and honestly, I had to prop-drill it all the way up to the server.js file to find out what it contains.

Reply:
oh, this is not intuitive to me. Hearing `label` I think of an axis-label or a popup-label, so something that is written-out and hence also human readable.

So what is the difference between `label` and `handle` in the example above? I think both are "mk", right?